### DISCUSSION TITLE
'The "Print Spooler" service was removed from the OS'

### DISCUSSION THREAD

**Discussion by techflashYT** [Sophia-Community/SophiApp#87] [General]
### Prerequisites

- [ ] Verify that this is not a Windows issue;
- [X] Refer to the [System requirements](https://github.com/Sophia-Community/SophiApp/blob/master/README.md#system-requirements);
- [X] I do not use a homebrew Windows image;
- [ ] I did not tweak Windows before that could cause system instability;
- [X] Attach a created log. You may find it in the `Logs` folder.

### Steps To Reproduce

1. Have already messed around with Windows to try to improve performance (One of the things I did was disable Print Spooler since I don't have a printer and have no need of any printing-based functionality)
2. Open SophiApp
3. Error

### Expected behavior

It doesn't give an error because the service is only used for printing as far as I know.

### Actual behaviour

It gives an error and I can't click on anything.

### Windows Version

Windows 11 Home 21H2 Build 22000.739

### SophiApp version

1.0.0.71

### SophiApp release type

SophiApp

### Logs

Windows 11 Home 21H2 build 22000.739
Computer name: MICHAELWIN11PC
User: micha
User domain: MICHAELWIN11PC
User culture: English (United States)
User region: United States
App version: 1.0.0.71
App is release: True
App folder: "C:\Users\micha\Downloads\SophiApp\"
App localization: EN
App theme: DARK
Antivirus: Malwarebytes

05.07.2022 20:33:12 Debug mode is: False
05.07.2022 20:33:12 Active view is: Loading
05.07.2022 20:33:12 Advanced settings is visible: False
05.07.2022 20:33:12 The "UWP for all users" switch state is: UNCHECKED
05.07.2022 20:33:12 The OS conditions checkings started
05.07.2022 20:33:12 OsVersionCondition run result: True
05.07.2022 20:33:12 The next condition to be run: OsBuildVersionCondition
05.07.2022 20:33:12 OsBuildVersionCondition run result: True
05.07.2022 20:33:12 The next condition to be run: OsFilesNotCorruptedCondition
05.07.2022 20:33:12 OsFilesNotCorruptedCondition run result: True
05.07.2022 20:33:12 The next condition to be run: RebootRequiredCondition
05.07.2022 20:33:12 RebootRequiredCondition run result: True
05.07.2022 20:33:12 The next condition to be run: SingleInstanceCondition
05.07.2022 20:33:12 SingleInstanceCondition run result: True
05.07.2022 20:33:12 The next condition to be run: SingleAdminSessionCondition
05.07.2022 20:33:12 SingleAdminSessionCondition run result: True
05.07.2022 20:33:12 The next condition to be run: Win10TweakerCondition
05.07.2022 20:33:12 Win10TweakerCondition run result: True
05.07.2022 20:33:12 The next condition to be run: SycnexScriptCondition
05.07.2022 20:33:12 SycnexScriptCondition run result: True
05.07.2022 20:33:12 The next condition to be run: DefenderWarningCondition
05.07.2022 20:33:12 DefenderWarningCondition run result: True
05.07.2022 20:33:12 The next condition to be run: DefenderNotCorruptedCondition
05.07.2022 20:33:13 DefenderNotCorruptedCondition run result: True
05.07.2022 20:33:13 The next condition to be run: SpoolerNotCorruptedCondition
05.07.2022 20:33:13 SpoolerNotCorruptedCondition run result: False
05.07.2022 20:33:13 The next condition to be run: NewVersionCondition
05.07.2022 20:33:13 Active view is: SpoolerCorrupted
05.07.2022 20:33:13 It took 1 second(s) to check the OS conditions

**Comment by farag2** [comment]
Hi! Actually, this is not an issue. This is by design. And it won't be fixed neither for you, nor for anyone.
Your problem is that all of you think that removing any service (not disabling it or setting a manual type of start) may result in speeding up OS. This is a myth! you cannot speed it up at all. All you do is making harm to the OS. But I may compile a special version for you with this checking disabled.

**Reply by techflashYT** [reply]
Alright, thanks, btw I don't think it would give a "negligeable" speed boost.  But I have a very fairly low end system (bought it in 2017 with minimal upgrades since), and even the maybe 8 or so MB of RAM that could be saved just from Windows having another svchost.exe to keep track of is enough for me to consider it worth it.

**Comment by farag2** [comment]
[SophiApp.zip](https://github.com/Sophia-Community/SophiApp/files/9054078/SophiApp.zip)

**Comment by farag2** [comment]
Working?

**Reply by techflashYT** [reply]
Yup, sorry for the wait.
